src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,23 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String model;
```

Comment:
Consider adding an access modifier to the 'model' field to encapsulate the property properly. In Java, it is best practice to use 'private' to restrict the access.